go-controller/pkg/util/dpu_annotations.go

```diff
@@ -45,9 +45,10 @@ const (
 )
 
 type DPUConnectionDetails struct {
-	PfId      string `json:"pfId"`
-	VfId      string `json:"vfId"`
-	SandboxId string `json:"sandboxId"`
+	PfId         string `json:"pfId"`
+	VfId         string `json:"vfId"`
+	SandboxId    string `json:"sandboxId"`
+	VfNetdevName string `json:"vfNetdevName,omitempty"`
```

Comment:
Why VfNetdevName is needed in DPU connection details? I thought pfId + vfId would be sufficient to uniquely identify a VF rep on the DPU, no?

Reply:
@zshi-redhat the VFNetdevName is `not` for the ovnkube-node running on the DPU. it is for the ovnkube-node running on the DPU Host.  

For example: Say, the VF name is **enp1s0f0v4**. When it is assigned to the Pod, it gets renamed to `eth0`. Now, ovnkbue-node running on the DPU Host will have  to rename the `eth0` interface back to the original VF Name. The SR-IOV device plugin and Kubelet deal with PCI Address and they don't have any clue about the names. So, we will need to `save` the name somewhere.

Reply:
I noticed that you added the same VfNetdevName in the PodInterfaceInfo structure, my question is why we need it also in the DPU connection details? My understanding is we can restore the name for PodInterfaceInfo, no?

Reply:
I see the difference, podInterfaceInfo is only for cmdAdd, populated from pod annotation, but we need to get the device name from pod annotation again in cmdDel, podInterfaceInfo doesn't pass though to cmdDel